### PR TITLE
Using pytorch builtin avoids no-grad params

### PR DESCRIPTION
**Pull request by dpressel**
Running a dataset besides an4 I hit this, and got NoneType errors since params with no gradients arent filtered.  The pytorch utility torch.nn.utils.clip_grad_norm does this filtering internally, and once I replaced the clipping code with it, the exception went away

### CODE REVIEW

**Comment by SeanNaren** [comment]
Thanks so much!
